public/src/components/channelManagement/epicTests/epicTestVariantEditor.tsx

```diff
@@ -359,24 +372,79 @@ const EpicTestVariantEditor: React.FC<EpicTestVariantEditorProps> = ({
         />
       )}
 
-      {allowVariantCustomPrimaryCta && (
```

Comment:
have you tested this on the apple news epic? In it's config it has `allowVariantCustomPrimaryCta: false`. So removing this check might have unintended consequences